contracts/Zauction.sol

```diff
@@ -31,6 +32,16 @@ contract ZAuction is Initializable, OwnableUpgradeable {
     uint256 expireBlock
   );
 
+  event BuyNow(
```

Comment:
Consider renaming `BuyNow` to `DomainSold` as it's more of an action / verb / past tense sort of thing